requirements-test.txt

```diff
@@ -3,3 +3,4 @@ coveralls
 mock
 nose
 nose-parameterized
+openpyxl
```

Comment:
We may as well. None of the current unit tests actually call openpyxl, but it needed to be included as it's still imported from digutils.